site/content/docs/5.3/utilities/borders.md

```diff
@@ -67,6 +67,12 @@ Here is a list of these extra classes:
 
 Or modify the default `border-color` of a component:
 
+{{< design-callout-alert >}}
+The **second and third** border color variants should not be used because they do not respect the Orange Design System specifications.
```

Comment:
```suggestion
The **second and third** border color variants should not be used in this context because they do not respect the Orange Design System specifications.
```

Reply:
Result of the discussion on Teams / ODS-WEB-Boosted : 
Julien + LM : IMO it avoids confusion to let the color coming from Bootstrap because using red might be OK in terms of rendering, but the developers won't never apply the border utility manually on an input (can be misleading here). On top of that, still IMO, it's easier to maintain, and easier to explain in the callout message since all the examples are not compilant with the DSM (can be explained more easily).

IC : OK, then let's not change the email address input border for red (we keep the green border coming from Bootstrap) and the design callout message will concern the 3 examples.
=> I'm going to change the design callout message